### DISCUSSION TITLE
Auto width settings for ex. partners logo showcase

### DISCUSSION THREAD

**Discussion by dzienisz** [nolimits4web/swiper#6317] [Q&A]
### Clear and concise description of the problem

I would like to show partners logos on my website with the same hight but different width. I see I can't do it with this swiper :( 

### Suggested solution

I don't know exackly how we should it implement it or if it possible but I think it would help a lot of people.

### Alternative

Maybe additional mode or new component?

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by dzienisz** [comment]
After some research I found that maybe this particular swiper won't solve all world (swiper) problems. I used different swiper that was exactly what I needed.

**Reply by sjmgarnier** [reply]
@dzienisz Would you mind sharing the swiper you used instead? I'm running into a similar issue.

**Comment by dzienisz** [comment]
https://splidejs.com/ it work great in that case!

Sent via Superhuman iOS ***@***.***>

**Reply by sjmgarnier** [reply]
Thanks!
